src/polycubed/src/rest_server.cpp

```diff
@@ -303,6 +305,10 @@ void RestServer::setup_routes() {
   Hateoas::addRoute("topology", base);
 
   router_->addNotFoundHandler(bind(&RestServer::redirect, this));
+
+  // prometheus metrics
```

Comment:
==> // add new endpoint to retrieve Prometheus-compatible metrics

Reply:
Thanks! :+1: